Week1

lemibk opened the pull request:
This is the first week of our project 
this part of the project contains the person,student and teacher classes in their respected files.
in those files we have implemented the concept of inheritance, constructor .etc

lemibk committed:
This the work for week one

lemibk committed:
this is week one

Benawi commented on Student.java:
```diff
@@ -0,0 +1,20 @@
+public class Student extends Person{
+    public int getClassroom() {
```
- Why do you create this method? not requested

Benawi commented on Student.java:
```diff
@@ -0,0 +1,20 @@
+    public int getClassroom() {
+        return classroom;
+    }
+    public void setClassroom(int classroom) {
```
- Why do you create this method? not requested

Benawi commented on Student.java:
```diff
@@ -0,0 +1,20 @@
+        this.classroom = classroom;
+    }
+
+    private int classroom;
```
- you do need to specify modifier and int?

Benawi commented on Teacher.java:
```diff
@@ -0,0 +1,21 @@
+class Teacher extends Person {
+    public String getSpecialization() {
```
- Not requested to create setter and getter

Benawi commented on Teacher.java:
```diff
@@ -0,0 +1,21 @@
+    }
+
+    public void setSpecialization(String specialization) {
+        this.specialization = specialization;
```
Not requested to create setter and getter

Benawi reviewed (changes requested):
#### Hi @lemibk 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

![giphy](https://user-images.githubusercontent.com/25227022/169391888-8dd8108f-0d42-4e54-87cc-a3f9fd392dbc.gif)


## Highlights

✔️  Create Person Class 
✔️  Create Student Class 
✔️  Create Teacher Class 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. @benawi

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

lemibk committed:
Update Student.java

i was checking if the code was working by adding a main.java class thats why i added the setters and getters but i have fixed now

lemibk committed:
Update Teacher.java

i was checking if the code was right on another main.java i have fixed the error now.

lemibk committed:
Update Student.java

Benawi reviewed (approved):
Hi @lemibk 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

![ ](https://media4.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif)
## Highlights
✔️ Create Person Class
✔️ Create Student Class
✔️ Create Teacher Class
✔️ PR is well descriptive 👍 
✔️ Requested changes implemented 👍
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you  take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@benawi** in your question so I can receive the notification.

------

lemibk committed:
Update Teacher.java
